docs/index.mdx

```diff
@@ -1,9 +1,15 @@
 ---
 id: index
-title: Documentation
-sidebar_label: Home
+title: Temporal Platform Documentation
+sidebar_label: Documentation Home
+description: Explore Temporal's comprehensive documentation to  build, scale, and manage reliable, fault-tolerant workflows with Workflow-as-Code solutions.
```

Comment:
This is really nice. I assume we don't care about the double space between "to" and "build" but just in case:
```suggestion
description: Explore Temporal's comprehensive documentation to build, scale, and manage reliable, fault-tolerant workflows with Workflow-as-Code solutions.
```